app/models/order.rb

```diff
@@ -3,14 +3,14 @@ class Order < ApplicationRecord
 
   has_many :item_orders
   has_many :items, through: :item_orders
+  belongs_to :user
```

Comment:
Adding the `users_id` column to the `orders` table broke a lot of tests. 

@lkriffell I think you had a similar issue when you added the `merchant_id` column to the `users` table. I think you said you were able to avoid those failures by not including a `belongs_to` in the model. Is that right? I tried it here and could not get it to work properly.

Reply:
Yeah. The problem was that no matter what kind of user you were creating, it would always want you to enter a merchant_id, so the creation would fail. But I would think that every single order should belong to a user in your case